Answers.md

```diff
@@ -1,11 +1,16 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+React is a jvascript framework that renders a virtual DOM. DOM manipulation is problematic, and can cause bugs when things happen on the app. React takes objects and causes them to interact with eachother smoothly when things change on the app. 
 
 1.  What does it mean to _think_ in react?
+To think in React is to think about how the app works in realtion to the UI, and how the app changes as the user progresses through the app. 

```

Comment:
Thinking in react is about breaking the project down into reusable components, with the goal of keeping your code dry. Also, data flow is from top (parent) to bottom (children)